### PR TITLE
renamed and streamlined lanes in Fastfile

### PR DESCRIPTION
**Pull request by sebalopez**
* Minor changes to Fastlane configuration. Reduced duplicated code and improved comments.
* Included cocoapods in Gemfile for running pod install
* Updated Readme

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -13,209 +13,174 @@
...
+workspace             = 'ios-base.xcworkspace'                                # The xworkspace file name
+certificates_git_url  = 'git@github.com:rootstrap/ios-base-certificates.git'  # The repo to store and sync certs and provisioning profiles.
+team_id               = ENV["FASTLANE_TEAM_ID"]                               # The organization's team id 
+itc_team_id           = ENV["FASTLANE_ITC_TEAM_ID"]                           # App Store Connect team id
+distribute_external   = false                                                 # Change this to true if you want the build to be distributed to external testers as well.
+notify_testers        = false                                                 #
+skip_wait             = true                                                  # Don't wait for the build to process. If set to true, the changelog won't be set, distribute_external option won't work and no build will be distributed to testers.
+devices               = 'platform=iOS Simulator,OS=9.2'                       # If using test lane, replace with proper array of devices to run the tests on
```
Comment (pMalvasio) [inline fastlane/Fastfile:31]: Right now the ios-base minimum os target is 11.4 
You can change this to `OS=11.4` in order to match that configuration. 

We don't develop for poor people now (kind of) 😆 we still support SE gen 1 tho

File: fastlane/Fastfile [hunk 2]
```diff
@@ -13,209 +13,174 @@
...
+slack_url             = ''                                                    # Slack webhook url and channel name for sending notifications upon completion
+slack_channel         = '#ios-builds'                                         #
```
Comment (pMalvasio) [inline fastlane/Fastfile:33]: What happen if this channel doesn't exists? The action fails blocking the build or the message is never created and that's it?

**Review by pMalvasio — APPROVED** [review]

File: fastlane/Fastfile (hunk 1 above)
Comment (sebalopez) [reply fastlane/Fastfile]: done :)

File: fastlane/Fastfile (hunk 2 above)
Comment (sebalopez) [reply fastlane/Fastfile:33]: the message is not sent, but the build does not fail (there's a `fail_on_error` param set to false)

Comment (pMalvasio) [reply fastlane/Fastfile:33]: Awesome 💪

**Review by CamilaMoscatelli — APPROVED** [review]

**Review by mato2593 — APPROVED** [review]

**Review by fernandatoledo — APPROVED** [review]

**Review by germanStabile — APPROVED** [review]

**Pull request merged**